### PR TITLE
Transaction logs

### PR DESCRIPTION
**Pull request by arslanhashmi**
Linked #https://trello.com/c/3m9vkbSw/10-api-endpoint-to-fetch-all-the-bookings-data

### CODE REVIEW

File: maritime_maas/urls.py [hunk 1]
```diff
@@ -39,7 +39,8 @@
         "v1/schema/redoc/",
         SpectacularRedocView.as_view(url_name="schema"),
         name="redoc",
-    )
+    ),
+    path("v1/booking/list/", BookingListView.as_view(), name="booking_list")
```
Comment (pjlampil) [inline maritime_maas/urls.py]: Path should be `v1/bookings/list` instead of `v1/booking/list`

**Review by pjlampil — CHANGES_REQUESTED** [review]
One small tweak in the comments and I couldn't get any response if I was applying `start_date` or `end_date` filters by providing date in YYYY-MM-DD format 🤔

**Review by pjlampil — APPROVED** [review]
LGTM 👍 🚀

File: maritime_maas/urls.py (hunk 1 above)
Comment (arslanhashmi) [reply maritime_maas/urls.py]: Made it `v1/open/bookings/` as `v1/bookings/list` was getting conflict with router `booking` urls.

**Pull request merged**